site/source/user-manual/service-guides/lightning/zeus-lnbits.rst

```diff
@@ -0,0 +1,124 @@
+      :width: 40%
+      :alt: extensions-enable
+
+#. Click **OPEN** *or* **LndHub** under *Extensions*:
+
+   .. figure:: /_static/images/services/lnbits/lndhub-open.png
+      :width: 40%
+      :alt: lndhub-open
+
+#. Make sure the wallet you just created is selected below the two QR codes:
+
+   .. figure:: /_static/images/services/lnbits/lndhub-select-wallet.png
+      :width: 40%
+      :alt: lndhub-select-wallet
+
+#. Install `Zeus <https://zeusln.app/>`_.
+
+#. Open it up and click **SCAN NODE CONFIG**.
+
+    .. figure:: /_static/images/services/lnbits/scan-node-config.jpg
+        :width: 25%
+        :alt: scan-node-config
+
+#. *If you only want this wallet to be able to RECEIVE PAYMENTS, scan this QR code:*
+
+    .. figure:: /_static/images/services/lnbits/left-qr.png
+        :width: 40%
+        :alt: left-qr
+
+    *If you are happy for this wallet to be able to both receive and MAKE payments scan this QR code:*
+
+    .. figure:: /_static/images/services/lnbits/right-qr.png
+        :width: 40%
+        :alt: right-qr
+
+#. Once scanned, name the wallet if you wish then hit **SAVE NODE CONFIG**.
+
+    .. figure:: /_static/images/services/lnbits/save-node-config.jpg
+      :width: 25%
+      :alt: save-node-config
+
+#. Zeus will now connect to your node and you'll see this screen:
+
+    .. figure:: /_static/images/services/lnbits/new-wallet-screen-zeus.png
+       :width: 25%
+       :alt: new-wallet-screen-zeus
+
+    .. tip:: If it doesn't work, please manually restart the Zeus app.
+
+#. You can now add sats to this wallet in two ways. You can allocate sats within LNbits by clicking the **+** icon here:
+
+    .. figure:: /_static/images/services/lnbits/plus-icon.png
+      :width: 60%
+      :alt: plus-icon
+
+    Let's add 1000 satoshis:
+
+    .. figure:: /_static/images/services/lnbits/add-1000-sats.png
+     :width: 60%
+     :alt: add-1000-sats
+
+    .. figure:: /_static/images/services/lnbits/1k-sats.png
+      :width: 60%
+      :alt: 1k-sats
+
+    Alternatively you can simply hit **Request** in Zeus and receive by generating a standard lightning invoice:
+
+    .. figure:: /_static/images/services/lnbits/zeus-request.png
+      :width: 20%
+      :alt: zeus-request
+
+    .. note:: This will only work if your node has inbound liquidity. And you cannot send sats from the LND node LNbits is using as that is not a regular lightning payment - that is a reblanace.
+
+#. Once you have added sats, you can click on this button within Zeus and see your new balance:
+
+    .. figure:: /_static/images/services/lnbits/zeus-balance-button.png
+      :width: 20%
+      :alt: zeus-balance-button
+
+    .. figure:: /_static/images/services/lnbits/zeus-balance.png
+     :width: 20%
+     :alt: zeus-balance
+
+Congratulations! Zeus is set up and ready to use lightning via your own lightning node - furthermore it will only be able to use your node in the way LNbits allows it.
```

Comment:
```suggestion
Congratulations! Zeus is set up and ready to use lightning via your own lightning node - furthermore it will only be able to use your node in the way you allow it, via LNbits.
```